cerebrium/development/serve.mdx

```diff
@@ -0,0 +1,105 @@
+5. When you're done, you can stop the server by pressing `Ctrl+C` in the same terminal where you started the server.
+
+## Usage
+
+To start a served instance, first navigate to the root folder of your **cortex deployment**.
+
+Then, simply run the following command in your terminal:
+
+```bash
+cerebrium serve start
+```
+
+After running the `cerebrium serve start` command, you will see output in your terminal indicating that the server is starting up. This output will include a URL that you can use to access your deployment locally.
+
+Here are some additional commands that can be useful when working with `cerebrium serve`:
+
+### Stopping a served instance
+
+To stop a served instance, you can simply press `Ctrl+C` in the terminal where the server is running. This will gracefully shut down the server and terminate the served instance.
+
+If, for some reason, your session does not close gracefully, you can also use the following command to end a served instance:
+
+```bash
+cerebrium serve stop <<serve_id>>
+```
+
+### Testing inference on the served instance
+
+#### Using the built in CLI predict utility
+
+The goal of the `cerebrium serve` is to allow you to build and test your deployment as quickly as possible.
+To help with this, we've added a `cerebrium serve predict` command that allows you to send requests to your served deployment without having to leave your ide or terminal.
+
+To use the `cerebrium serve predict` command, first start a served instance using `cerebrium serve start` and then run `cerebrium serve predict` with the input data for your deployment as a json object.
+
+For example:
+
+```bash
+cerebrium serve predict '{"prompt": "this is my input data"}'
+```
+
+#### Using the local REST server
+
+Whn you run `cerebrium serve start`, the CLI will print a local rest server address that you can use to send requests to your deployment.
+
+To use the local server to test inference, you can send a **post** request to: `localhost:8000/predict`
+and the body of the request should be a json object with the input data for your deployment.
+
+Here is an example curl command:
+
+```bash
+curl -X POST http://localhost:8000/predict -H "Content-Type: application/json" --data '{"prompt": "this is my input data"}'
+```
+
+Since we are using a local api server, you don't need to worry about providing an api key or any other authentication details.  
+Otherwise, you can interact with this endpoint as you would with a normal cerebrium endpoint.  
+Your served instance will be running on port 8000 by default, however, you can change this by using the `--port` flag when you start a served instance
+
+Additionally, if you have more than one serve instance running, you can specify the id of the serve instance you want to send the request by using the following pattern: `localhost:8000/{serve_id}/predict` and we'll make sure the request is sent to the correct instance.
+
+<Note>
```

Comment:
Not neccessary. I think we rather want to show a GIF/screenshot of what it looks like. A user can then see the results in the gif

Reply:
@Katsie011 please can you add a gif in this section?

Reply:
Good suggestion 👍